scripts/ingress-router.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
You might want to not use `set -u`, because with it, the test for no arguments does not work as intended, and the script fails with `./ingress-router.sh: line 63: POSITIONAL_ARGS[@]: unbound variable`

```suggestion
set -eo pipefail
```